Project.toml

```
@@ -1,7 +1,7 @@
 name = "Oceananigans"
 uuid = "9e8cae18-63c1-5223-a75c-80ca9d6e9a09"
 authors = ["Climate Modeling Alliance and contributors"]
-version = "0.80.0"
+version = "0.80.1"
```

```suggestion
version = "0.80.0"
```
we still haven't registered v0.80.0; I am waiting for #2963 to do that